diffkemp/semdiff/result.py

```diff
@@ -53,6 +56,15 @@ def __init__(self, name, filename=None, line=None, callstack=None,
             self.diff_kind = diff_kind
             self.covered = covered
 
+        def callstack_to_string(self):
```

Comment:
I think the naming of this function may be bit confusing, since it does not convert the callstack to a string in-place, but rather returns its string representation. Something like `callstack_as_string` or `get_callstack_string` would be more clear.

Reply:
Thanks. I have created `__str__` method in `Callstack` class for getting string representation of the call stack, I have also created a method `as_str_with_rel_paths` which will return string representation of the call stack with removed prefix of file paths.